recipe/set_bld_opts.bat

```diff
@@ -63,7 +63,7 @@ set BLD_OPTS=%WIN64% ^
     PNG_LIB=%LIBRARY_LIB%\libpng.lib ^
     PROJ_FLAGS="-DPROJ_STATIC -DACCEPT_USE_OF_DEPRECATED_PROJ_API_H" ^
     PROJ_INCLUDE="-I%LIBRARY_INC%" ^
-    PROJ_LIBRARY=%LIBRARY_LIB%\proj_6_0.lib ^
+    PROJ_LIBRARY=%LIBRARY_LIB%\proj_6_1.lib ^
```

Comment:
@ocefpaf maybe the `proj` recipe should copy `proj_6_X.lib` etc to a `proj.lib` so we don't need to update the recipes each time the version changes...

Reply:
Yeah, that makes more sense.

Reply:
As long as it is a copy and not a move operation. Maybe raising the issue in PROJ would be a good step as well.